style.css

```diff
@@ -0,0 +1,775 @@
+    line-height: 48px;
+    width: 45%;
+    padding-bottom: 49px;
+  }
+
+  .catalog__shop a {
+    font-size: 20px;
+    line-height: 19px;
+  }
+
+  .catalog__subtitle {
+    font-size: 30px;
+    margin-right: 2%;
+  }
+
+  .catalog__menu {
+    justify-content: normal;
+  }
+
+  .catalog__item {
+    width: 34%;
+    height: 352px;
+  }
+
+  .catalog__item_1 {
+    margin-right: 0;
+  }
+
+  .catalog__item:nth-child(n+3) { display: none; }
+  .catalog__item:first-child { 
+    margin: 0 11.2% 0 11%;
+  }
+
+  .catalog__img {
+    margin: 4px 0 27px 10px;
+    width: 90%;
+  }
+
+  .catalog__link {
+    font-size: 20px;
+    line-height: 19px;
+  }
+
+  .catalog__wrap {
+    justify-content: normal;
+    width: 105%;
+  }
+
+  .catalog__price {
+    width: 35%;
+    font-size: 25px;
+    line-height: 58px;
+    margin-right: 27.7%;
+  }
+
+  .catalog__price_1 {
+    margin-right: 22.6%;
+  }
+
+  .catalog__scale {
+    width: 34%;
+  }
+
+  .catalog__scale button {
+    height: 20px;
+    width: 20px;
+  }
+
+  .catalog__round_black {
+    order: -1;
+  }
+
+  .catalog__round_1 {
+    order: -2;
+  }
+
+  .catalog__arrow {
+    padding-left: 2px;
+    width: auto;
+  }
+
+  .catalog__item_1:before {
+    display: none;
+  }
+
+  .video {
+    padding-top: 10px;
+  }
+}
+
+@media (max-width: 754px) {
+  .catalog__wrapper {
+    height: 355px;
+  }
+
+  .catalog__link {
+    font-size: 18px;
+    line-height: 16px;
+  }
+
+  .catalog__scale button {
+    height: 16px;
+    width: 16px;
+  }
+}
+
+@media (max-width: 692px) {
+  .catalog__title {
+    font-size: 30px;
+    line-height: 36px;
+    padding-bottom: 20px;
+  }
+
+  .catalog__wrapper {
+    margin-bottom: 0;
+  }
+
+  .catalog__shop a {
+    font-size: 16px;
+  }
+
+  .catalog__link {
+    font-size: 16px;
+  }
+
+  .catalog__price {
+    font-size: 21px;
+  }
+}
+
+@media (max-width: 463px) {
+  .navigation {
+    height: 55px;
+  }
+
+  .navigation__logo {
+    width: 27%;
+    top: 23.5px;
+    left: 13.2%;
+  }
+
+  .navigation__burger {
+    top: 27.5px;
+    right: 12.5%;
+  }
+
+  .navigation__burger div {
+    width: 41px;
+    height: 6px;
+    margin-bottom: 5px;
+  }
+
+  .catalog__wrapper {
+    height: 191px;
+    padding: 0;
+    margin-bottom: 0;
+    background-position-x: -62px;
+    background-size: 187%;
+  }
+
+  .catalog__title {
+    font-size: 20px;
+    line-height: 20px;
+    padding: 64px 0 13px 12.8%;
+  }
+
+  .catalog__shop {
+    padding-left: 13%;
+  }
+
+  .catalog__shop a {
+    font-size: 12px;
+    line-height: 11px;
+  }
+
+  .catalog__arrow {
+    width: 15%;
+  }
+
+  .catalog__subtitle {
+    font-size: 20px;
+    margin-top: 34px;
+    margin-bottom: 0;
+    height: 20px;
+  }
+
+  .catalog__menu {
+    justify-content: center;
+    margin-bottom: 0;
+    height: 310px;
+  }
+
+  .catalog__item:first-child {
+    margin: 0;
+    height: 250px;
+    width: 66%;
+  }
+
+  .catalog__item:nth-child(2) { display: none; }
+  .catalog__item {
```

Comment:
Codacy found an issue: [Expected selector ".catalog__item" to come before selector ".catalog__item:first-child" (no-descending-specificity)](https://app.codacy.com/gh/WebHeroSchool/Project.Axion/pullRequest?prid=5456827)